### ISSUE TITLE
Vuetify 2 loader for vite

### ISSUE THREAD

**Issue by GabrielHangor** [vuetifyjs/vuetify-loader#335]
Hey, currently migrating from vue-cli to vite. Is vuetify loader for vite and vhetify 2 even a thing?

**Comment by meytin1337** [comment]
I am also interested in this. Migrating to vue 3 is not possible for us right now but vite is atleast the first step in the right direction.

**Comment by GabrielHangor** [comment]
> I am also interested in this. Migrating to vue 3 is not possible for us right now but vite is atleast the first step in the right direction.

I am just using unplugin-vue-components with vuetify resolver, recently migrated

**Comment by meytin1337** [comment]
Yes I also use unplugin-vue-components and so far everything seems to work as expected.

**Comment by jackhuynh95** [comment]
Appreciated `unplugin-vue-components` . It will makes sense to work well

**Comment by azuy212** [comment]
Yes I have also tried `unplugin-vue-components` with VuetifyResolver but SASS variables doesn't work without this loader. Is there any way to configure it in vite config?

**Comment by azuy212** [comment]
> Take a look at [vitejs/vite#12566 (comment)](https://github.com/vitejs/vite/discussions/12566#discussioncomment-5418403) You can use `additionalData` to import and your scss variables

Just tried it and it worked perfectly! 
Thanks
